src/Attachment.tsx

```diff
@@ -22,23 +22,22 @@ const Youtube = (props: {
     embedId: string,
     autoPlay?: boolean,
     loop?: boolean
-}) => {
-    const embedUrlQuery = {
-        "modestbranding": 1,
-        "loop": props.loop ? 1 : 0,
-        "autoplay": props.autoPlay ? 1 : 0
-    }
-
-    const embedUrl = new URI()
-                    .domain(YOUTUBE_DOMAIN)
-                    .subdomain("")
-                    .port("")
-                    .segment(["embed", props.embedId])
-                    .search(embedUrlQuery)
-                    .toString();
-
-    return <iframe type="text/html" src={ embedUrl }></iframe>
-}
+}) =>
+    <iframe
```

Comment:
you can either do `foo => { return bar; }` or` foo => bar;` - they are equivalent;